### DISCUSSION TITLE
Welcome to svelte-navigator Discussions!

### DISCUSSION THREAD

**Discussion by mefechoel** [mefechoel/svelte-navigator#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by quantuminformation** [comment]
oh this is new

**Reply by mefechoel** [reply]
Yes, it's a new github thing... I thought I could give it a try ;)
